migrations/1651671712077_recipe_trasactions_table.js

```diff
@@ -0,0 +1,26 @@
+exports.up = async (knex) => {
+  // check extension is not installed
+  const [extInstalled] = await knex('pg_extension').select('*').where({ extname: 'uuid-ossp' })
+
+  if (!extInstalled) {
+    await knex.raw('CREATE EXTENSION "uuid-ossp"')
```

Comment:
is this the uuid module we use elsewhere?

Reply:
in another migration, but if I don't check, then migration fails with the error for uuid

Reply:
mmm in another migration we add it, but if I remove this then this would fail

Reply:
I removed this from my order table and it seems to be working ok